tests/functional/Command/TranslationsMigrateCommandTest.php

```diff
@@ -114,10 +142,10 @@ private function assertTranslationIsSame(array $expected, string $key): void
         self::assertSame($expected, Translation::getByKey($key)?->getTranslations());
     }
 
-    private function assertMatchesSnapshotCommandDisplay(): void
+    protected function getSanitizedCommandOutput(): string|array|null
```

Comment:
`$displayOutput` is a `string`, right? If so, `preg_replace()` will also return a `string`. 

```suggestion
    protected function getSanitizedCommandOutput(): string|null
```